course/tests/tests_exportcourse.py

```diff
@@ -17,3 +17,7 @@ def test_exportcourse(self):
     def test_check_course_does_not_exists(self):
         with self.assertRaisesMessage(CommandError, 'Course "645343" does not exist'):
             management.call_command('exportcourse', 645343, stdout=StringIO(), verbosity=3)
+
+    def test_argument_type(self):
```

Comment:
this argument type check is a default Django feature, not a feature of the tested code, hence we don't need to test this, it's already tested at Django probably.

I'll prepare a short MR know to show that type of tests we need more of here